numpy/core/multiarray.py

```diff
@@ -71,9 +71,9 @@
 
 
 @array_function_from_c_func_and_dispatcher(_multiarray_umath.empty_like)
-def empty_like(prototype, dtype=None, order=None, subok=None):
+def empty_like(prototype, dtype=None, order=None, subok=None, shape=None):
     """
-    empty_like(prototype, dtype=None, order='K', subok=True)
+    empty_like(prototype, dtype=None, order='K', subok=True, shape=None)
```

Comment:
I just noticed order and subok both default to None rather than the ('K' and True) values documentation proposes and other _like() functions have. Which one is the correct? It seems to me that the function definition should be order='K' and subok=True, am I correct?

Reply:
It's a little confusing, but the arguments for this function (the dispatcher) need to default to None. We actually have a test that verifies this.

Reply:
Should we then update the documentation to reflect the real defaults?